### PR TITLE
Update README with compile instructions

### PR DESCRIPTION
**Pull request by jackharrhy**
# What
- See title
- Add .gitignore with .swp files ignored for vim users
# Test
- Spelign n gramr check
- Instructions on how to compile seem to work properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,28 @@ The content for the site lives in the `content/` folder. Fork this repo and send
...
 - Anything you can think of
 
+Compiling site locally
+----------------------
+
+Install [Lektor](https://www.getlektor.com/)
+
+```bash
+# Clone the repo
+git https://github.com/jackharrhy/Website.git`
```
Comment (Gudahtt) [inline README.md]: It looks like there's a `clone` missing here

Comment (jackharrhy) [reply README.md]: 💯 

cbdda82ebacc7d6c97d6312857e0cf3895d2e4f9

**Comment by EpicWayne** [comment]
Can someone remove me from this, I don’t think I have access to my github
anymore.
Thanks,
Noah

**Comment by Gudahtt** [comment]
@EpicWayne I've just removed you as a member. If that doesn't work, I'm afraid we can't help you. You'll have to find a way to unsubscribe, either by gaining access to your account or filtering blocking the emails.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,28 @@ The content for the site lives in the `content/` folder. Fork this repo and send
...
 - Anything you can think of
 
+Compiling site locally
+----------------------
+
+Install [Lektor](https://www.getlektor.com/)
+
+```bash
+# Clone the repo
+git clone https://github.com/jackharrhy/Website.git`
```
Comment (nint8835) [inline README.md]: I'd suggest you change this so it points to the computer science society's copy of the repo, and also remove the trailing ` unless it does something that I'm not aware of.

Comment (jackharrhy) [reply README.md]: Developed this branch on a fork, and completely forgot it was a fork... 🤦‍♂️  
5e1f68e

**Comment by EpicWayne** [comment]
Thanks Mark!

**Pull request merged**